### DISCUSSION TITLE
[IntelliSense] Highlight responsive classes

### DISCUSSION THREAD

**Discussion by Filip1139** [tailwindlabs/tailwindcss#5706] [Ideas]
Hello,

I have problem with readability when I styling responsive and hoverable elements. Is any existing VSCode extension for highlight TailwindCSS responsive variants in code?

![obraz](https://user-images.githubusercontent.com/24529620/123534568-62b5bb80-d71e-11eb-922d-2b4be8c3cdc5.png)

**Comment by babakfp** [comment]
Hi
Did you highlight them using vscode config file? If so, can you send me the code, please?
Thanks

**Comment by ellreka** [comment]
Hi
Added the ability to highlight variants in [Tailwind CSS Highlight](https://github.com/ellreka/tailwindcss-highlight).
If you like, you can try it.

![visual-image2](https://user-images.githubusercontent.com/28590054/193499757-bcccc85d-7da5-4753-b7e8-2513853d970a.png)
